### DISCUSSION TITLE
Предложение по Playlist.getDescription

### DISCUSSION THREAD

**Discussion by sabaone** [Chimildic/goofy#107] [Идея] [answered]
Привет!
Во-первых, огромное спасибо создателю библиотеки! Недавно начал с ней работать, но впечатления самые хорошие.
Во-вторых, есть предложение по функции Playlist.getDescription - пусть она возвращает только имена исполнителей, без фразы "и не только". К примеру, у меня описание на английском, и эта фраза смотрится странно.

**Comment by Chimildic** [comment, marked as answer]
Текущая реализация `Playlist.getDescription` была больше для примера, совпадая с описанием от Spotify в некоторых плейлистах. 
Можно пересмотреть, добавить разные варианты конечно. Но они не охватят всех сценариев. Поэтому лучше сделать свое.

По принципу [инструкции](https://github.com/Chimildic/goofy/discussions/18) переопределить, чтобы не потерять при обновлениях
```js
Playlist.getDescription = function (tracks, limit = 5) {
    let artists = Array.from(new Set(tracks.map(t => t.artists[0].name)));
    return Selector.sliceRandom(artists, limit).join(', ');
}
```

**Reply by sabaone** [reply]
Спасибо, работает
